evennia/server/portal/discord.py

```diff
@@ -215,12 +236,14 @@ def start(self):
             # get the gateway URL from Discord
             self.get_gateway_url()
         else:
+            # set the retry flag so we maintain this connection
+            self.do_retry = True
             connectWS(self)
 
 
 class DiscordClient(WebSocketClientProtocol, _BASE_SESSION_CLASS):
     """
-    Implements the grapevine client
```

Comment:
I remember seeing this in the review and thinking "No worries, I'll fix that after merge" and then completely forgetting about it :D